Gemfile

```diff
@@ -55,7 +55,8 @@ gem 'whenever', require: false
 # Dependencies
 gem 'adal', git: 'git@github.com:moneyadviceservice/azure-activedirectory-library-for-ruby'
 gem 'cream', '2.1.7'
-gem 'dough-ruby', '~> 5.35.0'
+gem 'dough-ruby', '~> 5.36.1'
```

Comment:
I think this should be 5.36

Reply:
This limits the updates to:
`">= 5.36.1 and < 5.37.0"`

So it will only update at `patch` level

Reply:
I've changed this to `~> 5.36`